front/src/config/i18n/fr.json

```diff
@@ -715,6 +715,58 @@
         "defaultDeletionError": "Une erreur s'est produite lors de la suppression de l'appareil.",
         "conflictError": "L'appareil actuel est déjà dans Gladys."
       }
+    },
+    "yeelight": {
+      "title": "Yeelight",
+      "description": "Contrôler les lumières Yeelight.",
+      "deviceTab": "Appareils",
+      "discoverTab": "Découverte Yeelight",
+      "discoverDeviceDescr": "Scanner automatiquement les appareils Yeelight",
+      "nameLabel": "Nom de l'appareil",
+      "namePlaceholder": "Entrez le nom de votre appareil",
+      "modelLabel": "Modèle",
+      "ipLabel": "Adresse IP",
+      "roomLabel": "Pièce",
+      "saveButton": "Sauvegarder",
+      "updateButton": "Mettre à jour",
+      "alreadyCreatedButton": "Déjà créé",
+      "deleteButton": "Supprimer",
+      "unmanagedModelButton": "Modèle non pris en charge",
+      "device": {
+        "title": "Appareils Yeelight dans Gladys",
+        "search": "Chercher les appareils",
+        "noDevices": "Aucun appareil Yeelight n'a encore été ajouté.",
+        "deviceOnNetworkTitle": "Appareils détectés sur le réseau",
+        "scanButton": "Recherche sur le réseau",
+        "noDeviceFound": "Aucun appareil Yeelight trouvé.",
+        "noDevicesFound": "Aucun appareil Yeelight trouvé. Assurez-vous d'avoir activé le \"LAN Control\".",
+        "connectButton": "Connecter/Reconnecter",
+        "deviceNotHandled": "L'appareil n'est pas encore géré, veuillez nous contacter pour nous aider à le connecter dans Gladys !",
+        "nameLabel": "Nom de l'appareil",
+        "namePlaceholder": "Entrez le nom de votre appareil",
+        "noNameLabel": "No name",
+        "modelLabel": "Modèle",
+        "ipLabel": "Adresse IP",
+        "roomLabel": "Pièce",
+        "featuresLabel": "Fonctionnalités",
+        "noFeatures": "Aucune fonctionnalité",
+        "saveButton": "Sauvegarder",
+        "editButton": "Modifier",
+        "deleteButton": "Supprimer"
+      },
+      "discover": {
+        "title": "Appareils détectés sur le réseau",
+        "description": "Les appareils Yeelight configurés sont automatiquement découverts sur le réseau LAN. Vos appareils Yeelight doivent avoir le \"LAN Control\" activé. En savoir plus <a href=\"https://documentation.gladysassistant.com/en/configuration#yeelight-lan-control\">ici</a>.",
```

Comment:
The link is not the same as provided in the documentation here: https://github.com/GladysAssistant/v4-website/pull/19/files !